### PR TITLE
Flatten out pointing and add example

### PR DESCRIPTION
**Pull request by dakota002**

### CODE REVIEW

**Comment by dakota002** [comment]
@jracusin , @Vidushi-GitHub can either of you provide a description for the attitude_quarternion?

File: gcn/notices/core/Pointing.schema.json [hunk 1]
```diff
@@ -6,46 +6,43 @@
...
+      "description": "Rotation angle [deg] between the spacecraft X-Z plane and the plane defined by the spacecraft X-axis and celestial North"
+    },
+    "attitude_quarternion": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 4,
+      "maxItems": 4,
+      "description": ""
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:32]: Representing attitude state of a spacecraft, using four element vector: a scalar and 3 element unit vector.

Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:32]:
```suggestion
      "description": "Representing attitude state of a spacecraft, using four element vector: a scalar and 3 element unit vector."
```

File: gcn/notices/core/Pointing.schema.json [hunk 2]
```diff
@@ -6,46 +6,43 @@
...
+      "description": "Rotation angle [deg] between the spacecraft X-Z plane and the plane defined by the spacecraft X-axis and celestial North"
+    },
+    "attitude_quarternion": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 4,
+      "maxItems": 4,
+      "description": "Representing attitude state of a spacecraft, using four element vector: a scalar and 3 element unit vector."
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:32]:
```suggestion
      "description": "Representing attitude state of a spacecraft, using four element vector [q1, q2, q3, q4], composed of a scalar and 3 element unit vector."
```

File: gcn/notices/core/Pointing.schema.json [hunk 3]
```diff
@@ -6,46 +6,43 @@
...
+    "attitude_quarternion": {
+      "type": "array",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 4,
+      "maxItems": 4,
+      "description": "Representing attitude state of a spacecraft, using four element vector [q1, q2, q3, q4], composed of a scalar and 3 element unit vector."
+    },
+    "rotation": {
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:34]:
```suggestion
    "rotation_rate": {
```

File: gcn/notices/core/Pointing.schema.json [hunk 4]
```diff
@@ -6,46 +6,43 @@
...
+      "type": "number",
+      "description": "RA pointing of the instrument Z axis"
+    },
+    "dec": {
+      "type": "number",
+      "description": "Dec pointing of the instrument Z axis"
+    },
+    "roll": {
+      "type": "number",
+      "description": "Roll of the instrument about its Z axis"
```
Comment (Vidushi-GitHub) [inline gcn/notices/core/Pointing.schema.json:19]:
```suggestion
      "description": "Roll [deg] of the instrument about its Z axis"
```

**Pull request merged**